docs/apache-airflow/tutorial.rst

```diff
@@ -102,11 +102,31 @@ define a ``schedule_interval`` of 1 day for the DAG.
     :start-after: [START instantiate_dag]
     :end-before: [END instantiate_dag]
 
+Operators
+---------
+
+An operator defines a unit of work for Airflow to complete. Using operators is the classic approach
+to defining work in Airflow. For some use cases, it's better to use the TaskFlow API to define
+work in a Pythonic context as described in :doc:`/tutorial_taskflow_api`. For now, using operators helps to
+illustrate the upstream and downstream dependencies between tasks.
```

Comment:
```suggestion
work in a Pythonic context as described in :doc:`/tutorial_taskflow_api`. For now, using operators helps to
visualize task dependencies in our DAG code.
```
Is this wording better? Honestly not sure

Reply:
I like it.